Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -201,6 +201,15 @@ struct PerWebProcessState {
 
 #endif // PLATFORM(IOS_FAMILY)
 
+#if HAVE(UIFINDINTERACTION)
+enum FindOverlayView {
```

Comment:
```suggestion
enum class FindOverlayViewPosition : uint8_t {
```

Probably best to also move this to the `.mm` file, since it isn't needed outside of it.